README.md

```diff
@@ -48,6 +52,28 @@ class A
   methodA: ->
 ```
 
+#### With Overrides
+
+```json
+"alphabetize_keys": {
+  "module": "coffeelint-alphabetize-keys",
+  "overrides": ["methodC", "methodB", "methodA"]
+}
+```
+```coffee
```

Comment:
Looks good to me. Please just move this beneath the explanation of the key categories